app/src/main/java/io/capibaras/abcall/viewmodels/LoginViewModel.kt

```
@@ -4,48 +4,109 @@ import androidx.compose.runtime.getValue
+                                val jsonObject = JSONObject(errorBody)
+                                jsonObject.getString("message")
+                            } catch (e: Exception) {
+                                ErrorUIState.Error(R.string.error_authenticate)
+                            }
+                        } else {
+                            ErrorUIState.Error(R.string.error_authenticate)
+                        }
+                        ErrorUIState.Error(R.string.error_authenticate)
+                    }
+                }
+            } catch (e: Exception) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)